### PR TITLE
update grafana panels for TiKV GC

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5324
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -222,20 +222,18 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
-- GC speed：GC 每秒删除的 key 的数量
+- TiDB GC worker actions：TiDB GC worker 的不同 action 的个数
 - TiKV AutoGC Working：Auto GC 管理器的工作状态
 - ResolveLocks Progress：GC 第一阶段（ResolveLocks）的进度
 - TiKV Auto GC Progress：GC 第二阶段的进度
+- GC speed：GC 每秒删除的 key 的数量
 - TiKV Auto GC SafePoint：TiKV GC 的 safe point 的数值，safe point 为当前 GC 的时间戳
 - GC lifetime：TiDB 设置的 GC lifetime
 - GC interval：TiDB 设置的 GC 间隔
+- GC in Compaction Filter：write CF 的 Compaction Filter 中已过滤版本的数量
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:236]: @hicqu PTAL

Comment (hicqu) [reply grafana-tikv-dashboard.md:236]: LGTM

**Review by hicqu — APPROVED** [review]

**Pull request merged**